templates/rhpam73-authoring.yaml

```diff
@@ -38,146 +38,146 @@ parameters:
   value: myapp
   required: true
 - displayName: KIE Admin User
-  description: KIE administrator username
+  description: KIE administrator username.
   name: KIE_ADMIN_USER
   value: adminUser
   required: false
 - displayName: KIE Admin Password
-  description: KIE administrator password
+  description: KIE administrator password.
   name: KIE_ADMIN_PWD
   from: "[a-zA-Z]{6}[0-9]{1}!"
   generate: expression
   required: false
 - displayName: KIE Server Controller User
-  description: KIE server controller username (Sets the org.kie.server.controller.user system property)
+  description: KIE server controller username. (Sets the org.kie.server.controller.user system property)
   name: KIE_SERVER_CONTROLLER_USER
   value: controllerUser
   required: false
 - displayName: KIE Server Controller Password
-  description: KIE server controller password (Sets the org.kie.server.controller.pwd system property)
+  description: KIE server controller password. (Sets the org.kie.server.controller.pwd system property)
   name: KIE_SERVER_CONTROLLER_PWD
   from: "[a-zA-Z]{6}[0-9]{1}!"
   generate: expression
   required: false
 - displayName: KIE Server Controller Token
-  description: KIE server controller token for bearer authentication (Sets the org.kie.server.controller.token system property)
+  description: KIE server controller token for bearer authentication. (Sets the org.kie.server.controller.token system property)
   name: KIE_SERVER_CONTROLLER_TOKEN
   required: false
 - displayName: KIE Server User
-  description: KIE server username (Sets the org.kie.server.user system property)
+  description: KIE server username. (Sets the org.kie.server.user system property)
   name: KIE_SERVER_USER
   value: executionUser
   required: false
 - displayName: KIE Server Password
-  description: KIE server password (Sets the org.kie.server.pwd system property)
+  description: KIE server password. (Sets the org.kie.server.pwd system property)
   name: KIE_SERVER_PWD
   from: "[a-zA-Z]{6}[0-9]{1}!"
   generate: expression
   required: false
 - displayName: KIE Server Bypass Auth User
-  description: KIE server bypass auth user (Sets the org.kie.server.bypass.auth.user system property)
+  description: Allow API requests to KIE server wuthout user authentication. (Sets the org.kie.server.bypass.auth.user system property)
```

Comment:
"wuthout" should be "without"

Reply:
I think it should be changed to:

"Allows the KIE server to bypass the authenticated user for task related operations e.g. queries. (Sets the org.kie.server.bypass.auth.user system property)"

Gotten from https://docs.jboss.org/drools/release/7.18.0.Final/drools-docs/html_single/